psh/tools/psh.py

```diff
@@ -35,9 +35,20 @@ def _readable(exp_regex):
     return exp_regex.replace(PROMPT, '(psh)% ')
 
 
-def assert_cmd(pexpect_proc, cmd, expected='', msg='', is_regex=False):
+def _check_result(pexpect_proc, result):
+    if result == 'dont-check':
+        return
+
+    if (result != 'success') and (result != 'fail'):
+        raise Exception(f''.join((f'The value {result} for `result` argument is not correct. ',
+            'Please choose between susccess/fail/dont-check')))
+
+    assert_cmd_successed(pexpect_proc) if result == 'success' else assert_cmd_failed(pexpect_proc)
+
+
+def assert_cmd(pexpect_proc, cmd, expected='', result='success', msg='', is_regex=False):
```

Comment:
Some tests are failing now as msg is passed as positional argument. When adding new argument putting it on the end is usually the safest option.

To avoid this probem altogether [keword-only arguments](https://peps.python.org/pep-3102/) can be used  
Note that python has special syntax to disallow using arguments positionally:
```suggestion
def assert_cmd(pexpect_proc, cmd, *, expected='', result='success', msg='', is_regex=False):
```
Calling this function with more than 2 positional arguments will raise an Exception. Additionaly there is simillar way to force arguments to be positional using `/` as an argument.